components/Article/SeriesNavButton.js

```diff
@@ -90,12 +89,31 @@ const SeriesNavButton = ({ t, series, router }) => {
         }}
       >
         <span {...styles.title}>
+          {series.logo && (
+            <>
+              <img
+                {...styles.logo}
+                src={series.logo}
```

Comment:
```js
import { imageResizeUrl } from 'mdast-react-render/lib/utils'
```

For the case that SVGs are not used: we should resize to 48 height (2x of actual height for retina):
```suggestion
                src={imageResizeUrl(series.logo, 'x48')}
```

The asset server will ignore resize instructions for SVG so we do not need to check here.